components/docs-chef-io/content/habitat/pattern-library.md

```diff
@@ -22,6 +22,10 @@ For help with Chef Habitat and these patterns, ask:
 - In the [Chef Habitat Slack](http://slack.habitat.sh/)
 
 +++
+[\[edit on GitHub\]](https://github.com/habitat-sh/habitat/blob/master/components/docs-chef-io/content/habitat/pattern-library.md)
+
+**Note:** If you are using a container scheduler such as Kubernetes, Swarm, or Mesos DC/OS, or a PaaS such as Cloud Foundry, you should not follow the bastion ring pattern, because the scheduler handles persistence and orchestration on your behalf.
+
```

Comment:
This note (and the one later on) are confusing given that the section on the very next line is about the Kubernetes Bastion Ring Pattern. Is there a distinction that we're trying to create here, or did I miss something?

Reply:
The message is "Don't use a container scheduler with a bastion ring". Is this still accurate?